spajoa.py

```diff
@@ -52,8 +52,7 @@ def run_optimal_737(objective = 'fuel'):
     m = Model(m.cost, BCS(m))
     m_relax = relaxed_constants(m, None, ['M_{takeoff}', '\\theta_{db}'])
 
-    sol = m_relax.localsolve(verbosity=0, iteration_limit=50, reltol=0.01,
-                             modifylastgp=True)
+    sol = m_relax.localsolve(verbosity=0, iteration_limit=50, reltol=0.01)
     post_process(sol)
```

Comment:
@1ozturkbe email any udpated plots to @pgkirsch , he has to give them individual files